warzone/OrdersDriver.h

```diff
@@ -5,4 +5,5 @@
 
 namespace OrdersDriver {
 	int main();
+	int oldMain();
```

Comment:
Deleto

Reply:
Just left it in so I could reference but will be sure to delete before final PR